### ISSUE TITLE
Subscriptions: Commented out code in KarParserImpl

### ISSUE THREAD

**Issue by medhost-chara** [drajer-health/eCRNow#838]
Are subscriptions still supported? 

Re commented out code:
https://github.com/drajer-health/eCRNow/blob/926729099d38e35831c402294f7f2179a86cf586/src/main/java/com/drajer/bsa/service/impl/KarParserImpl.java#L471C73-L471C92

**Comment by nbashyam** [comment]
This is commented out because we have not been able to find an EHR that supports creation of Subscription Topics and notifications. If Medhost is ready we can get this uncommented and test it out with a sandbox access.

**Comment by medhost-chara** [comment]
@nbashyam  should the code look for this boolean to identify if it should or should not?
https://github.com/drajer-health/eCRNow/blob/master/src/main/java/com/drajer/bsa/model/HealthcareSetting.java#L283

**Comment by nbashyam** [comment]
Yes, you can uncomment and test it out using the flag iin the Healthcare Setting.
